app.config.js

```diff
@@ -51,6 +51,10 @@ module.exports = function (config) {
     : undefined
   const UPDATES_ENABLED = !!UPDATES_CHANNEL
 
+  const SENTRY_DIST = `${PLATFORM}.${VERSION}.${DIST_BUILD_NUMBER ?? ''}${
```

Comment:
Can DIST_BUILD_NUMBER actually be not present? seems weird to have `1.2.`

Reply:
Probably can remove it, since we don't ever release under the same 1.X more than once.

Reply:
Actually, that might not be a good idea. We upload Sentry symbols on every build, and I don't know if Sentry will overwrite old symbols.

Reply:
I mean more, does it have to have `?? ''`?

Reply:
I'd at least expect `DIST_BUILD_NUMBER ?? 0`